Dockerfile

```diff
@@ -10,12 +10,19 @@ ARG QEMU_REPO
 WORKDIR /src
 RUN git clone $QEMU_REPO && \
   cd qemu && \
-  git checkout $QEMU_VERSION && \
-  scripts/git-submodule.sh update \
+  git checkout $QEMU_VERSION
+
+WORKDIR /src/qemu
+COPY patchs /tmp
+ARG BINARY_PREFIX
+RUN if [ -n "$BINARY_PREFIX" ]; then patch --verbose -p1 < /tmp/shebang-fix.patch; fi
```

Comment:
I don't think this is a good idea. It means that the qemu version is not configurable anymore as these patches (eventually) will not apply on certain versions. Also unclear how to update/verify the correctness.

Reply:
I'm actually bringing patches back with #43 so I guess we can reconsider it later. Also people are reporting positive feedback with `cpu=max` so might need to make it default as well.